docs/bokeh/source/docs/user_guide/basic/areas.rst

```diff
@@ -13,7 +13,7 @@ Rectangles
 Bars
 ~~~~
 
-.. TODO: refs to bars
+See :ref:`ug_basic_bars`
```

Comment:
This is OK but in case you are in interested to improve this further, the main thing to get across about bars is that their specify are a center/span in one dimension, and coordinates in the other dimension. 

All the "rect-like" glyphs basically differ in this way, e.g. `rect` is center and span in both dimensions, `quad` is coordinates in both dimensions, blocks are a corner and a span in both dimensions, etc. 

Actually I guess ideally there might even be a table that summarizes all the rect-like glyphs this way.